osu.Game/OsuGame.cs

```diff
@@ -992,7 +994,7 @@ private void showOverlayAboveOthers(OverlayContainer overlay, OverlayContainer[]
                 overlay.Depth = (float)-Clock.CurrentTime;
         }
 
-        private void forwardLoggedErrorsToNotifications()
+        private void forwardRuntimeLogsToNotifications()
```

Comment:
Good catch. This was not supposed to be the case in previous code since logs would be posted to a custom logger name, but now it has changed to post to a defined logging target (`Input`). Will update the conditions.